### DISCUSSION TITLE
Console font size as independent setting / aligned to Terminal font size

### DISCUSSION THREAD

**Discussion by francisbarton** [posit-dev/positron#4399] [Feedback]
Currently in Positron, the font size in the console tracks the font size in the editor (editor.fontSize).
It doesn't change with the setting for the Terminal font (terminal.integrated.fontSize).

In VSCodium, by contrast, if I change the Terminal font size setting, this also affects the size in the R console (which suits me fine). I was surprised that this behaviour was set up differently in Positron.

Would it be possible to switch this behaviour, so that the R console font size tracks the terminal font size instead - or even has its own independent setting?

To me it makes more sense to have the console font size track the terminal font size, as they tend to be found in the same pane of the UI.

**Comment by juliasilge** [comment]
Thanks for this feedback! 🙌 I agree with you and have opened https://github.com/posit-dev/positron/issues/4411 to track and prioritize this work.
